packages/taxdat/R/stan_input_helpers.R

```diff
@@ -2131,6 +2131,19 @@ get_loctime_combs <- function(stan_data) {
   # Set names for identification
   names(loctime_combs) <- u_obs
   
+  # for multi-year observations, separate the loctime. 
+  if(any(sapply(loctime_combs,length)>1)){
+    loctime_combs_tmp <- loctime_combs[sapply(loctime_combs,length)>1]
+    fixed_loctime_combs<-list()
```

Comment:
This seems ok but related to our discussion of how R uses memory, the following would technically be more memory-efficient:

`fixed_loctime_combs <- as.list(rep(NA, length(loctime_combs_tmp)))`

and then down below instead of line 2141...

`fixed_loctime_combs[[row_idx]] <- loctime_combs_tmp2`